### ISSUE TITLE
Question: Could we have a chatbot ui to interact with model

### ISSUE THREAD

**Issue by alabulei1** [LlamaEdge/search-api-server#2]
Could we interact with the model with the search feature via the Chatbot UI, just as we did in the llama-api-server.wasm?

**Comment by suryyyansh** [comment]
Yes, it works the same way. Simply drop chatbot-ui in the root folder.
